setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='hm_pyhelper',
-    version='0.6.1',
+    version='0.6.3',
```

Comment:
This had to be bumped twice because of this: https://github.com/NebraLtd/hm-pyhelper/issues/20